external-servers/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  dnsmasq:
+    build: ./dnsmasq
+    privileged: true
+    environment:
+      - TERM=xterm-256color
+    volumes:
+      - ./test_packets:/test_packets
```

Comment:
This volume isn't required, but if you want to pass persistent data back and forth between dnsmasq for debugging you can